components/file_system/src/lib.rs

```diff
@@ -87,6 +90,39 @@ impl IOType {
     }
 }
 
+#[derive(Clone, Copy)]
+pub struct IOContext {
+    pub(crate) io_type: IOType,
+    /// Total I/O bytes used by current thread.
+    pub(crate) total_read_bytes: usize,
+    /// Buffered I/O bytes used after `total_read_bytes` is refreshed, by
```

Comment:
Can't understanding with only reading these comments

Reply:
total_read_bytes is disk IO, and it's only refreshed periodically. Here records the buffered IOs not accounted by total_read_bytes.